cmd/copy.go

```diff
@@ -217,18 +218,50 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 	}
 
 	// Note: new implementation of list-of-files only works for remove command and upload for now.
-	if cooked.fromTo == common.EFromTo.BlobTrash() || cooked.fromTo == common.EFromTo.FileTrash() || cooked.fromTo.From() == common.ELocation.Local() {
-		cooked.listOfFilesLocation = raw.listOfFilesToCopy
-	} else if cooked.fromTo == common.EFromTo.BlobFSTrash() {
-		// Note: when the ADLS Gen2 interop happens eventually, we should keep the current blob implementation
-		// so that users can still use include/exclude flags,
-		// and we can shift to the new directory delete API (equivalent to the current bfs implementation)
-		// if no include/exclude is supplied.
-		if len(raw.include) > 0 || len(raw.exclude) > 0 {
+	if fromTo.To() == common.ELocation.Unknown() || cooked.fromTo.From() == common.ELocation.Local() {
+		// This handles both list-of-files and include-path as a list enumerator.
```

Comment:
BTW, all the in-app help comments (in sync, remove and copy) all still say that wildcards are supported in include-path and exclude-path. Can you correct that please?

Reply:
Corrected.